src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
-| T2 weighted        | T2w              |                                                                                                                                                   |
-| T1 Rho map         | T1rho            | Quantitative T1rho brain imaging <br> <https://www.ncbi.nlm.nih.gov/pubmed/24474423> <br> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346383/> |
-| T1 map             | T1map            | quantitative T1 map                                                                                                                               |
-| T2 map             | T2map            | quantitative T2 map                                                                                                                               |
-| T2\*               | T2star           | High resolution T2\* image                                                                                                                        |
-| FLAIR              | FLAIR            |                                                                                                                                                   |
-| FLASH              | FLASH            |                                                                                                                                                   |
-| Proton density     | PD               |                                                                                                                                                   |
-| Proton density map | PDmap            |                                                                                                                                                   |
-| Combined PD/T2     | PDT2             |                                                                                                                                                   |
-| Inplane T1         | inplaneT1        | T1-weighted anatomical image matched to functional acquisition                                                                                    |
-| Inplane T2         | inplaneT2        | T2-weighted anatomical image matched to functional acquisition                                                                                    |
-| Angiography        | angio            |                                                                                                                                                   |
+        sub-<label>[_ses-<label>][_acq-<label>][_part-<label>][_echo-<index>][_fa-<index>][_inv-<index>][_mt-<on/off>][_ce-<label>][_rec-<label>][_run-<index>]_<suffix>.nii[.gz]
+        sub-<label>[_ses-<label>][_acq-<label>][_part-<label>][_echo-<index>][_fa-<index>][_inv-<index>][_mt-<on/off>][_ce-<label>][_rec-<label>][_run-<index>][_mod-<suffix>]_defacemask.nii[.gz]
```

Comment:
I wonder if `fa-` entity has a danger of confusion with fractional anisotropy, which seems likely to end up being a suffix (though it may be capitalized to `FA`). Would `flip-` be clear? Or could it potentially clash with some other concept?

Reply:
Relying on the case to differentiate 'fa' from 'FA' seems very unsafe...
Now the 'FA' derived from DWI data will appear as a suffis, indicating the nature of the map, while the 'fa-' here is an indexable key appearing in the filename. So what is the actual risk of confusing them?

Reply:
I need to check back whether we've explicitly excluded overlap of suffix and entity in the past. If we haven't done that, then it's probably not a problem, but I wanted to flag this while I was thinking about it.

Reply:
How about being explicit and replacing `fa-` with `excitationflipangle-` and `-FA` with `fractionalanisotropy-`. Is there a limit  to the number of characters for BIDS filenames?

Reply:
No limit, but that would definitely annoy people.

Reply:
Ok then. I don't think `flip-` wouldn't be clear. How about `angle-`?

Reply:
Most manuscripts use FA for flip angle (and for fractional anisotropy).

Reply:
Also, is there a compiled database of terms already used in BIDS (or in the planning stage) that we could search through, to make sure that we're not using any terms that another BEP is working on? Or maybe a parameter/flag/etc name booking system for BIDS development? For example I know QSM and T2* imaging can be head orientation dependent, so they might be planning to use the term `angle` already? But going through all BEPs manually isn't really feasible to make sure.

Reply:
I agree with @ChristophePhillips, and I think we can rely on that amount of heuristic.

Reply:
@mathieuboudreau, I think it is a very hard and convoluted task. I would say 'first come - first served'. :)

Reply:
@mathieuboudreau #423 and #466 are pushing in that direction. I'm not sure what the status of each is.

Reply:
@tiborauer that makes sense. If that's the case, I'd vote for FA = flip angle and hope this BEP gets merged before the diffusion one does? If diffusion gets merged before this BEP, then we could just switch it to `flip-` or `angle-` then.

Reply:
I think both `_FA` and `fa-` can live in one spec, as @ChristophePhillips pointed out.

Reply:
Hey everybody, apologies for being silent on this issue for so long. It's a tricky question and we want to set a good precedent for future BEPs, so we have been discussing this in the maintainers group and want to also discuss with the steering group to be clear about our options. The next steering group meeting is on the 30th, so it'll take some more time to resolve this ... but it's important to get this right.

Reply:
Glad it's a priority and you're working on it! It would be unfortunate to introduce another naming issue inconstant with the MRI literature (see: "TR" problem)

Reply:
Per yesterday's call, the BEP team will change `fa-<index>` to `flip-<index>`, so I think I'll resolve this thread. Please reopen if I've done so prematurely.